pkg/store/bucket.go

```diff
@@ -1364,15 +1364,18 @@ func (s *BucketStore) Series(req *storepb.SeriesRequest, seriesSrv storepb.Store
 					"block.resolution": blk.meta.Thanos.Downsample.Resolution,
 				})
 
-				if err := blockClient.ExpandPostings(sortedBlockMatchers, seriesLimiter); err != nil {
-					span.Finish()
-					return errors.Wrapf(err, "fetch series for block %s", blk.meta.ULID)
-				}
 				onClose := func() {
 					mtx.Lock()
 					stats = blockClient.MergeStats(stats)
 					mtx.Unlock()
 				}
+
+				if err := blockClient.ExpandPostings(sortedBlockMatchers, seriesLimiter); err != nil {
+					onClose()
+					span.Finish()
+					return errors.Wrapf(err, "fetch series for block %s", blk.meta.ULID)
```

Comment:
Can we name it `expand postings` or `fetch postings`?
`fetch series` might be a little bit confusing here

Reply:
Yes, this was there before. Changed to fetch postings